src/lib/authui/moreauthcomps/edit_realm_dialog.rb

```diff
@@ -0,0 +1,176 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE Linux GmbH.
+#
+# ------------------------------------------------------------------------------
+
+require 'yast'
+require 'auth/authconf'
+require 'authui/moreauthcomps/generic_input_dialog'
+Yast.import 'UI'
+Yast.import 'Icon'
+Yast.import 'Label'
+
+module SSSD
```

Comment:
If this class is named  `SSSD::EditRealmDialog` then its file should be `sssd/edit_realm_dialog.rb` instead of 
`moreauthcomps/edit_realm_dialog.rb`.